core/conventional-commits/__tests__/conventional-commits.test.js

```diff
@@ -333,6 +333,44 @@ describe("conventional-commits", () => {
       expect(rootChangelogContent).toMatchSnapshot("root");
     });
 
+    it("should contain default header when custom header is not passed", async () => {
```

Comment:
this test is redundant, the default value has already been asserted many times with snapshots